app/controllers/teams_controller.rb

```diff
@@ -48,7 +47,7 @@ def create
   def update
     @conferences = conference_list
     respond_to do |format|
-      if @team.update_attributes(team_params)
+      if @team.update_attributes(team_params) && terms_accepted?
```

Comment:
Same as for the orga controller - I think this can be solved cleaner via an acceptance validation.

In addition to that, orga people should not really need to check the two terms-checkboxes anyways. But as said, that's an extra thing, not super necessary for now.